plugin.json

```diff
@@ -62,7 +62,15 @@
                 "help_text": "(Optional) Set to lock the plugin to a single Bitbucket organization.",
                 "placeholder": "",
                 "default": null
+            },
+            {
+                "key": "BitbucketSelfHostedUrl",
+                "display_name": "Bitbucket Self-hosted Url",
+                "type": "text",
+                "help_text": "(Optional) Set to change the default bitbucket URL to a self-hosted one.",
```

Comment:
```suggestion
                "help_text": "If using Bitbucket Server instead of Bitbucket Cloud, please set this value to your BitBucket server's URL.",
```

Reply:
Changed! thank you!